optional/gabz_config.lua

```diff
@@ -0,0 +1,1783 @@
+Config = {}
```

Comment:
consider putting a comment at the top explaining the differences between this and the regular config. Also explain that the fxmanifest would need to be updated to point to this config if someone wants to use it.